tools/failures.py

```diff
@@ -11,14 +11,18 @@
 
 import seta
 
-def getRawData():
-    url = "http://alertmanager.allizom.org/data/seta/"
+def getRawData(startDate, endDate):
+    if not startDate and not endDate:
```

Comment:
Not sure if this would actually be used, but if endDate is not specified it could default to today.